gammapy/datasets/map.py

```diff
@@ -2342,47 +2344,255 @@ def to_image(self, name=None):
         energy_axis = self._geom.axes["energy"].squash()
         return self.resample_energy_axis(energy_axis=energy_axis, name=name)
 
-    def peek(self, figsize=(12, 8)):
-        """Quick-look summary plots.
+    def peek(self, figsize=(13.0, 7)):
+        """Quick-look summary plots for a given MapDataset:
+        - Exposure map
+        - Counts map
+        - Npred map
+        - Exposure profile
+        - Energy dispersion matrix
+        - PSF containment radius
 
         Parameters
         ----------
         figsize : tuple
-            Size of the figure. Default is (12, 10).
+            Size of the figure. Default is (13.5, 7).
 
         """
 
+        def plot_counts(ax, counts_data, cmap, vmin, vmax):
+            """
+            Plot the counts map on a given axis.
+
+            Parameters
+            ----------
+            ax : `matplotlib.axes.Axes`
+                The matplotlib Axes to plot on.
+            counts_data : `Map`
+                Gammapy Map containing the counts data.
+            cmap : Colormap
+                The colormap to use for plotting.
```

Comment:
For an inline function, parameter description is not useful. If the title is clear enough, even the docstring is not necessary

Reply:
done